test/fixtures/code_samples/main.php

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+<body>
+
+<h1>Developer News</h1>
+
+<?php echo "The Best PHP Examples"; ?>
+
+// This is a single-line comment
+
+# This is also a single-line comment
+
+</body>
+</html> 
```

Comment:
Could you also add a block comment anywhere on this file ? Actually I prefer if It's only php code, could you modify to only have php code ?